bin/lib/installation.py

```diff
@@ -1078,6 +1134,8 @@ def _targets_from(node, enabled, context, name, base_config):
     'nightlytarballs': NightlyTarballInstallable,
     'nightly': NightlyInstallable,
     'script': ScriptInstallable,
+    'solidity': SolidityInstallable,
```

Comment:
was wondering if this name is appropriate, but seems so, its specific to the .json file and the solidity_available_releases() function